packages/server/src/healthcheck.ts

```diff
@@ -1,16 +1,33 @@
 import { MEDPLUM_VERSION } from '@medplum/core';
 import { Request, Response } from 'express';
+import { getInstanceId } from './cloud/aws/metadata';
 import { getDatabasePool } from './database';
+import { setGauge } from './otel/otel';
 import { getRedis } from './redis';
 
 export async function healthcheckHandler(_req: Request, res: Response): Promise<void> {
+  const instanceId = getInstanceId();
+
+  setGauge('medplum.db.idleConnections', getDatabasePool().idleCount, { instanceId });
+  setGauge('medplum.db.queriesAwaitingClient', getDatabasePool().waitingCount, { instanceId });
+
+  let startTime = Date.now();
+  const postgresOk = await testPostgres();
+  const dbRoundtripMs = Date.now() - startTime;
+  setGauge('medplum.db.healthcheckRTT', dbRoundtripMs / 1000, { instanceId });
+
+  startTime = Date.now();
+  const redisOk = await testRedis();
+  const redisRoundtripMs = Date.now() - startTime;
+  setGauge('medplum.redis.healthcheckRTT', redisRoundtripMs / 1000, { instanceId });
+
```

Comment:
Nice, clean 👍

Reply:
What does "RTT" stand for?

Reply:
Roundtrip time, if it's not clear we can unabbreviate 🙃